src/ch04-03-slices.md

```diff
@@ -236,7 +236,7 @@ a slice that contains a pointer to the 7th byte (counting from 1) of `s` with a
 ここで、`starting_index`はスライスの最初の位置、`ending_index`はスライスの終端位置よりも、
 1大きい値です。内部的には、スライスデータ構造は、開始地点とスライスの長さを保持しており、
 スライスの長さは`ending_index`から`starting_index`を引いたものに対応します。以上より、
-`let world = &s[6..11];`の場合には、`world`は`s`の（1から数えて）7バイト目へのポインタと5という長さを保持するスライスになるでしょう。
+`let world = &s[6..11];`の場合には、`world`は`s`の（1から数えて）6バイト目へのポインタと5という長さを保持するスライスになるでしょう。
```

Comment:
「（1から数えて）」があると意味が変わってしまいます。現在の英語の原文に合わせて、以下のような訳に変更してもらえないでしょうか？

現在の英語の原文：
> `world` would be a slice that contains a pointer to the byte at index 6 of s with a length value of 5.

変更の提案：

```suggestion
`let world = &s[6..11];`の場合には、`world`は`s`の添え字6のバイトへのポインタと5いう長さを持つスライスになるでしょう。
```

* * *

**詳細**

この和訳を作成したときの英語の原文は以下のようになっており、「（1から数えて）7バイト目」という訳自体は正しいです。
> `world` would be a slice that contains a pointer to the 7th byte (counting from 1) of `s` with a length value of 5.

しかし、その後、原文の方でも「1から数えて」の部分が混乱を招くという指摘があったようで、以下のように変更されました。
> `world` would be a slice that contains a pointer to the byte at index 6 of `s` with a length value of 5.

- コミット：rust-lang/book [Remove ordinal numbers and only refer to indexes to avoid confusion #219b9b6e](https://github.com/rust-lang/book/commit/219b9b6e97ea8621d1406d3d37acfdc5214c30ce)

Reply:
理解できました。経緯までありがとうございます！頂いた変更に更新しました。